plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3954,6 +3957,38 @@ public void testPathHiddenColumn()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testFileModifiedTimeHiddenColumn()
+    {
+        ZonedDateTime beforeTime = (ZonedDateTime) computeScalar("SELECT current_timestamp(3)");
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_file_modified_time_", "(col) AS VALUES (1)")) {
+            MaterializedResult expectedColumns = resultBuilder(getSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                    .row("col", "integer", "", "")
+                    .build();
+            MaterializedResult actualColumns = computeActual("DESCRIBE " + table.getName());
+            // Describe output should not have the $file_modified_time hidden column
+            assertEquals(actualColumns, expectedColumns);
+
+            ZonedDateTime fileModifiedTime = (ZonedDateTime) computeScalar("SELECT \"$file_modified_time\" FROM " + table.getName());
+            ZonedDateTime afterTime = (ZonedDateTime) computeScalar("SELECT current_timestamp(3)");
+            assertThat(fileModifiedTime).isBetween(beforeTime, afterTime);
+        }
+    }
+
+    @Test
+    public void testDeleteWithFileModifiedTimeColumn()
+    {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_delete_with_file_modified_time_", "(key int)")) {
+            assertUpdate("INSERT INTO " + table.getName() + " VALUES (1)", 1);
```

Comment:
`(1)` -> `1` 

The paranthesis are not actually needed in this case.

Reply:
I knew, but I believe most test cases use paranthesis in INSERT and there's no such consensuns.